src/main/java/io/vertx/ext/mongo/GridFSInputStream.java

```diff
@@ -1,54 +0,0 @@
-package io.vertx.ext.mongo;
-
-import com.mongodb.reactivestreams.client.gridfs.AsyncInputStream;
-import io.vertx.core.AsyncResult;
-import io.vertx.core.Handler;
-import io.vertx.core.buffer.Buffer;
-import io.vertx.core.streams.WriteStream;
-import io.vertx.ext.mongo.impl.GridFSInputStreamImpl;
-
-/**
- * {@link GridFSInputStream} which bridges the gap between vertx's {@link WriteStream}
- * and mongodb's {@link AsyncInputStream}.
- * <p>
- * Adapted from com.github.sth.vertx.mongo.streams.GridFSInputStream
- *
- * @author <a href="https://github.com/st-h">Steve Hummingbird</a>
- */
-public interface GridFSInputStream extends AsyncInputStream, WriteStream<Buffer> {
```

Comment:
Was there a reason for this to be public? I have the impression it was only useful internally.

Reply:
I don't know but its gone now :)

Reply:
I don't mind removing, I was just curious why it was made public in the first place, as it looks like it's only useful internally.